tests/ci/unit_tests_staging.yml

```diff
@@ -0,0 +1,91 @@
+# unit_tests_staging.yml

```

Comment:
is the plan to have one yaml for each pipeline? 

the other day we were talking about having the same pipeline for staging and master. Can this be merged?

Reply:
done.